templates/redis.conf.erb

```diff
@@ -66,8 +66,8 @@ bind <%= @bind_arr.join(' ') %>
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-<% if @unixsocket %>unixsocket <%= @unixsocket %><% end %>
-<% if @unixsocketperm %>unixsocketperm <%= @unixsocketperm %><% end %>
+<% unless @unixsocket.empty? %>unixsocket <%= @unixsocket %><% end %>
+<% unless @unixsocketperm.empty? %>unixsocketperm <%= @unixsocketperm %><% end %>
```

Comment:
The `unixsocketperm` should pop in the file only if `unixsocket` is not empty.
By this way, the user of the module does have to take care of `unixsocketperm` when `unixsocket` is disabled.

Reply:
I like that idea.